test_scripts/Smoke/Resumption/commonStepsResumption.lua

```diff
@@ -0,0 +1,71 @@
+local commonStepsResumption = {}
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+
+
+local function CheckTimeBoundaries(time)
+  local timeout = tonumber(commonFunctions:read_parameter_from_smart_device_link_ini("ApplicationResumingTimeout"))
+  return function (exp, _)
+    if exp.occurences == 2 then
+      local time2 = timestamp()
+      local time_to_resumption = time2 - time
+      if time_to_resumption >= timeout and time_to_resumption < (timeout + 500) then
+        commonFunctions:userPrint(33, "Time to HMI level resumption is " .. tostring(time_to_resumption) ..", expected ~ " .. timeout )
+        return true
+      else
+        commonFunctions:userPrint(31, "Time to HMI level resumption is " .. tostring(time_to_resumption) ..", expected ~ " .. timeout )
+        return false
+      end
+    elseif exp.occurences == 1 then
+      return true 
+    end
+  end
+end
+
+function commonStepsResumption:RegisterApp(app)
+  local correlation_id = Test.mobileSession:SendRPC("RegisterAppInterface", app)
+  local exp = EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered",
+  						{ application = {appName = app.appName}, resumeVrGrammars = true }):Do(function (_, data)
+    app.hmi_app_id = data.params.application.appID
+  end)
+  Test.mobileSession:ExpectResponse(correlation_id, { success = true})  
```

Comment:
@LitvinenkoIra , please add verification for OnHMIStatus(NONE). For more details see requirement: "[HMI Status] [GENIVI]: SDL must assign NONE HMILevel for all registered apps independently on whether app from HMILevel-resumption list or not "